kml/serdes_common.hpp

```diff
@@ -20,9 +20,10 @@ uint32_t ToRGBA(Channel red, Channel green, Channel blue, Channel alpha)
          static_cast<uint8_t>(blue) << 8 | static_cast<uint8_t>(alpha);
 }
 
-std::string PointToString(m2::PointD const & org);
+std::string PointToString(m2::PointD const & org, char const separator);
 
-std::string PointToString(geometry::PointWithAltitude const & pt);
+std::string PointToString(geometry::PointWithAltitude const & pt, char const separator = ',');
```

Comment:
nits: it's better to avoid implicit parameters in such cases. Explicit params are very clear in the place of their usage.

An even clearer approach is something like this:

```
void SomeFunction()
{
  // ...
  char constexpr kLineStringCoordinateSeparator = ',';
  PointToString(pt, kLineStringCoordinateSeparator);
  // ...
}
```

Reply:
I removed the implicit parameter and created 2 funcs with explicit separators that compose explicit type of string, because the PointToLineString is used in 4 different places and passing the parameter every time may not be clear:
```cpp
std::string PointToLineString(geometry::PointWithAltitude const & pt)
{
  char constexpr separator = ',';
  if (pt.GetAltitude() != geometry::kInvalidAltitude)
    return PointToString(pt.GetPoint(), separator) + separator + strings::to_string(pt.GetAltitude());
  return PointToString(pt.GetPoint(), separator);
}

std::string PointToGxString(geometry::PointWithAltitude const & pt)
{
  char constexpr separator = ' ';

...

```